router/core/operation_processor.go

```diff
@@ -749,12 +753,25 @@ func (o *OperationKit) normalizePersistedOperation(clientName string, isApq bool
 }
 
 type NormalizationCacheEntry struct {
-	operationID              uint64
 	normalizedRepresentation string
 	operationType            string
 	operationDefinitionRef   int
 }
 
+type VariablesNormalizationCacheEntry struct {
+	normalizedRepresentation string
+	uploadsMapping           []uploads.UploadPathMapping
+	id                       uint64
+	variables                json.RawMessage
+	reparse                  bool
+}
+
+type RemapVariablesCacheEntry struct {
+	internalID               uint64
+	normalizedRepresentation string
+	remapVariables           map[string]string
+}
+
```

Comment:
Can we get a few comments and descriptions here to understand the purpose and the fields?